examples/lkj.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+import torch
+
+import pyro
+import pyro.distributions as dist
+import torch.distributions.constraints as constraints
+import math
+from pyro.infer.mcmc import HMC, MCMC, NUTS
+from pyro import poutine
+
+"""
+This simple example is intended to demonstrate how to use an LKJ prior with
+a multivariate distribution.
+"""
+
+pyro.enable_validation(True)
```

Comment:
Thanks for adding an example! Would you mind wrapping this in a `main()` function with `argparse` and options for `--num-samples` and `--warmup-steps` (see [examples/baseball.py](https://github.com/pyro-ppl/pyro/blob/dev/examples/baseball.py#L309)). This is a little more work, but it allows us to [test examples](https://github.com/pyro-ppl/pyro/blob/dev/tests/test_examples.py#L19) and to add configuration for `--jit` and `--cuda`.

Reply:
Yes, I will do so, thanks.

Reply:
Example should be conformant now.